python/tests/geom.py

```diff
@@ -349,6 +349,29 @@ def check_warnings(sim, should_warn=True):
         sim.add_flux(15, 6, 2, fregion)
         check_warnings(sim)
 
+    def test_rotate(self):
+
+        e_sus = [mp.LorentzianSusceptibility(sigma_diag=mp.Vector3(1, 2, 3),
+                                             sigma_offdiag=mp.Vector3(12, 13, 14)),
+                 mp.DrudeSusceptibility(sigma_diag=mp.Vector3(1, 2, 3),
+                                        sigma_offdiag=mp.Vector3(12, 13, 14))]
+
+        mat = mp.Medium(epsilon_diag=mp.Vector3(1, 2, 3), epsilon_offdiag=mp.Vector3(12, 13, 14),
+                        E_susceptibilities=e_sus)
+
+        rot_matrix = mp.Matrix(mp.Vector3(1, 0, 0), mp.Vector3(0, 0, 1), mp.Vector3(0, -1, 0))
```

Comment:
It would be better to use a [rotation matrix](https://en.wikipedia.org/wiki/Rotation_matrix#Basic_rotations) with off-diagonal components (e.g., clockwise rotation about the *z* axis):

```py
import math

rot_angle = math.radians(23.9)
rot_matrix = mp.Matrix(mp.Vector3(math.cos(rot_angle),math.sin(rot_angle),0), mp.Vector3(-math.sin(rot_angle),math.cos(rot_angle),0), mp.Vector3(0,0,1))
```

Reply:
Yours is fine. (It would have been good to use a rotation matrix with non-zero elements everywhere but this will suffice.)

Reply:
I see what you're saying. I'll add such a test.